src/Controller/EnMarche/SummaryManagerController.php

```diff
@@ -219,6 +221,23 @@ public function unpublishAction()
     }
 
     /**
+     * @Route("/competences/autocompletion",
+     *     name="app_summary_manager_skills_autocomplete",
+     *     condition="request.isXmlHttpRequest()"
+     * )
+     * @Method("GET")
+     */
+    public function skillsAutocompleteAction(Request $request)
+    {
+        $this->disableInProduction();
+
+        $skills = $this->getDoctrine()->getRepository(Skill::class)
+            ->findAvailableSkillsForAdherent(trim(strip_tags($request->get('term'))), $this->getUser());
```

Comment:
Instead of `trim(strip_tags($request->get('term')))`, use `$slugifier->slugify($request->query->get('term'))`